### DISCUSSION TITLE
Should 'State' column be renamed?

### DISCUSSION THREAD

**Discussion by lylebrown** [kromitgmbh/titra#124] [Ideas]
I was looking through the code and wiki for information on how the Siwapp integration works, and discovered how it works through the `useState` global setting. This is tied to the "State" column in the timecard Details page, which is displayed if the "Use time entry state?" option in Administration is enabled.

This wasn't immediately apparent to me from the name "State," and I didn't associate invoicing with that Administration setting at first. Could we consider changing the column name to "Invoice State" or similar, and the "Use time entry state?" Administration setting to reflect that change (such as "Use time entry invoicing?")?

I would also accept a rejection of this change if you believe the term makes enough sense on its own - I will admit that I haven't dabbled much in timekeeping or billing software until now, and I could just be a bit dense. 😄

**Comment by faburem** [comment]
Hi and thanks for the feedback! What do you think about renaming it to "Billing state"?

**Reply by lylebrown** [reply]
I think that would be descriptive enough to work!
